Tools/CISupport/ews-build/steps.py

```diff
@@ -1312,9 +1339,18 @@ def validate_github(self, pr_number):
 
         repository_url = self.getProperty('repository', '')
 
-        pr_json = self.get_pr_json(pr_number, repository_url)
-        if not pr_json:
+        for index in range(self.GITHUB_RETRY + 1):
```

Comment:
why `+ 1`? index is already zero based.

also index -> `attempt`

Reply:
+1 is so `retry=0` will still make the first request, at least